python/example1/example1.py

```diff
@@ -68,13 +68,16 @@ def parse_json( str ):
 # Create socket
 sock = socket.socket(socket.AF_INET)
 # Create a SSL context with the recommended security settings for client sockets, including automatic certificate verification
-context = ssl.create_default_context()
-# Alternatively, a customized context could be created
-#context = ssl.SSLContext(ssl.PROTOCOL_SSLv23)
-#context.verify_mode = ssl.CERT_REQUIRED
-#context.check_hostname = True
+#context = ssl.create_default_context()
+# the folowing lines require Python 3.7+ and OpenSSL 1.1.0g+ to specify minimum_version
+context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+context.options |= ssl.OP_NO_TLSv1
+context.options |= ssl.OP_NO_TLSv1_1
+context.minimum_version = ssl.TLSVersion.TLSv1_2
+context.verify_mode = ssl.CERT_REQUIRED
+context.check_hostname = True
 # Load a set of default CA certificates from default locations
-#context.load_default_certs()
+context.load_default_certs()                                              
```

Comment:
We should still be able to use `create_default_context()` here. The only new behavior we need vs before is setting the `minimum_version`.